docs/reference/how-to/recipes/stemming.asciidoc

```diff
@@ -228,3 +228,5 @@ In the above case, since `ski` was in-between quotes, it was searched on the
 `body.exact` field due to the `quote_field_suffix` parameter, so only document
 `1` matched. This allows users to mix exact search with stemmed search as they
 like.
+Note that if the choice of field passed in `quote_field_suffix` does not exist 
```

Comment:
Maybe use a `NOTE:` for this ? Can you also add the same note in the `query_string` doc  ?

Reply:
> Can you also add the same note in the query_string doc 

 I can add a comment if  you like but `quote_field_suffix` is only mentioned there in a list of short param descriptions which feels like the wrong place to describe an edge case:

<img width="396" alt="Query_String_Query___Elasticsearch_Reference__7_1____Elastic" src="https://user-images.githubusercontent.com/170925/59271261-5d769880-8c4b-11e9-9b0f-6127c9f167b2.png">
It already offers a direct link to this page for detailed info.

Reply:
Ok, fine with me